dfsnew/docker/docker_test.go

```diff
@@ -0,0 +1,233 @@
+// Copyright 2015 The Serviced Authors.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build integration
+
+package docker
+
+import (
+	"bytes"
+	"testing"
+
+	dockerclient "github.com/fsouza/go-dockerclient"
+	. "gopkg.in/check.v1"
+)
+
+func TestDocker(t *testing.T) { TestingT(t) }
+
+type DockerSuite struct {
+	dc     *dockerclient.Client
+	docker *DockerClient
+	regid  string
+}
+
+var _ = Suite(&DockerSuite{})
+
+func (s *DockerSuite) SetUpSuite(c *C) {
```

Comment:
The point of this docker library is to make sure we are getting expected behavior from the docker client, so that other packages can interact with it accordingly.  I don't know if it necessarily makes sense to write unit tests on something we don't necessarily own.